src/main/java/com/minecolonies/coremod/blocks/cactus/BlockCactusSlabDouble.java

```diff
@@ -0,0 +1,52 @@
+package com.minecolonies.coremod.blocks.cactus;
+
+import com.minecolonies.api.util.constant.Constants;
+import com.minecolonies.coremod.blocks.AbstractBlockSlab;
+import com.minecolonies.coremod.creativetab.ModCreativeTabs;
+import net.minecraft.block.BlockStoneSlab;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.block.BlockStoneSlab'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)